zulipterminal/ui_tools/views.py

```diff
@@ -589,3 +590,107 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
         if is_command_key('GO_BACK', key) or is_command_key('HELP', key):
             self.controller.exit_pop_up()
         return super(HelpView, self).keypress(size, key)
+
+
+class MsgInfoView(urwid.ListBox):
+    def __init__(self, controller: Any, msg: Any) -> None:
+        self.controller = controller
+        self.msg = msg
+
+        if msg['type'] == 'stream':
+            recipient = msg['display_recipient'] + " -> " + msg['subject']
+        elif not msg['display_recipient'][0]['full_name'] ==\
+                msg['sender_full_name']:
+            recipient = msg['display_recipient'][0]['full_name']
+        else:
+            recipient = 'Self'
```

Comment:
I'm not sure about this private-message logic:
* while perhaps obvious, we've generally included 'you' (or 'self' here) within the list of recipients we display
* this doesn't seem to work for multi-recipient PMs

Reply:
I think we should remove the `Recipient` part from message info, since we are very clearly showing it in normal view.

Reply:
I think removing it makes sense after #347. @neiljp, what do you say?